exploration/expression-attributes.md

```diff
@@ -36,20 +36,62 @@ At least the following expression attributes should be considered:
 
 - Attributes with a formatting runtime impact:
 
-  - `fallback` — A value to use instead of the default fallback,
-    should the expression's primary formatting fail in some way.
+  - `id` — An identifier for the expression.
+    This is included in the formatted part,
+    and allows the parts of an expression to be explicitly addressed.
+
+    > Example identifying two literal numbers:
+    >
+    > ```
+    > The first number was {1234 :number @id=first} and the second {56789 :number @id=second}.
+    > ```
+
   - `locale` — An override for the locale used to format the expression.
     Should be expressed as a non-empty sequence of BCP 47 language codes.
+
+    > Example embedding a French literal in an English message:
+    >
+    > ```
+    > In French, “{|bonjour| @locale=fr}” is a greeting
+    > ```
+
   - `dir` — An override for the LTR/RTL/auto directionality of the expression.
 
+    > Example explicitly isolating the directionality of a placeholder:
+    >
+    > ```
+    > Welcome, {$username @dir=auto}
+    > ```
+
 - Attributes relevant for translators, tools, and other message operations,
   but with no runtime impact:
 
   - `example` — A literal value representing
     what the expression's formatted value will look like.
+
+    > Example providing a translator with an example for `$place`:
+    >
+    > ```
+    > Hello {$place @example=world}
+    > ```
+
   - `note` — A comment on the expression for translators.
+
+    > Example providing a translator with an example for `$place`:
+    >
+    > ```
+    > Welcome to {$place @note=|The user's current location|}
+    > ```
+
   - `translate` — A boolean `yes`/`no` indicator communicating to translators
     whether the expression should or should not be localised.
```

Comment:
Add a note about the relationship to e.g. ITS/HTML's `translate` attribute, which also uses `yes`/`no` rather than `true`/`false`?

Reply:
```suggestion
  - `translate` — A boolean `yes`/`no` indicator communicating to translators
    whether the expression should or should not be localised.
    The values here correspond to those used for this property in HTML and elsewhere.
```